ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/private/cucushift-installer-rehearse-azure-ipi-private-workflow.yaml

```diff
@@ -0,0 +1,30 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-ipi-private
+  steps: 
+    pre:
+    - ref: azure-provision-resourcegroup
+    - ref: azure-provision-vnet
+    - ref: azure-provision-nat
+    - ref: azure-provision-bastionhost
+    - ref: proxy-config-generate
+    - ref: ipi-conf
+    - ref: ipi-conf-azure
+    - ref: ipi-conf-azure-provisionednetwork
+    - ref: ipi-conf-azure-proxy
+    - ref: ipi-install-monitoringpvc
+    - ref: ipi-install-rbac
+    - ref: openshift-cluster-bot-rbac
+    - ref: ipi-install-install
```

Comment:
this install step will check if sourcing proxy script or not [1] 
[1] https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L214-L218